spacy/lang/fr/lemmatizer.py

```diff
@@ -13,7 +16,9 @@ class FrenchLemmatizer(Lemmatizer):
     the lookup table.
     """
 
-    def __call__(self, string, univ_pos, morphology=None):
+    def __call__(
+        self, string: str, univ_pos: str, morphology: Optional[Morphology] = None
```

Comment:
`morphology` is a dict of morphological features, not a `Morphology` object.

Reply:
Ah, damn, I copy-pasted that around so this is everywhere now! I'll just do a search and replace!

Reply:
It's not a great name and I'm not even sure the format is really standardized.

Reply:
I can probably change it to a `MorphAnalysis` after this is merged, but for now it should be a `Dict`.